### PR TITLE
made a page AboutMe

### PR DESCRIPTION
**Pull request by NosatskyiK**
Added title text, main text and main image on page about me. Added mobile a version

Acceptance criteria for 1

- [x]  Mentor is added to pr
- [x]  AC copied to pr description
- [x]  link to deployed version added to pr
- [x]  About block (text and image) added to the main page
- [x]  On the widths 390px and 1280px, the site layout corresponds to the design

Acceptance criteria for 1*

- [x]  Created separate component About
- [x]  Its styles added to [.module.css](https://github.com/css-modules/css-modules)
- [ ]  For App.tsx styles moved to App.module.css

P.S. Маю певні сумніви щодо правильності виконання останнього пункту про перенос стилів до App.module.css, я створив окремі css файл для компоненту і всі стилі знаходяться там, а в App.css прописав тільки декілька обнуляючих стилів. Якщо зробив все таки не правильно, то можу пофіксити якщо трішки поясните цей пункт. Дякую)

Link - https://nosatskyik.github.io/fe-react-2024/

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,71 +1,71 @@
-:root {
```
Comment (vadym-zinchenko-moc) [inline src/index.css:1]: why you commented styles? if you don't use this maybe need to delete?

File: src/components/AboutMe/aboutMe.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  font-weight: 400;
+  line-height: 32px;
+}
+
+.link {
+  color: inherit;
+  font-weight: inherit;
+}
+
+@media (max-width: 390px) {
```
Comment (vadym-zinchenko-moc) [inline src/components/AboutMe/aboutMe.css]: the mobile version does not match the design
Screenshot 2024-04-21 at 22 08 12

**Review by vadym-zinchenko-moc — CHANGES_REQUESTED** [review]

**Comment by NosatskyiK** [comment]
Resized for mobile devices and added style modules, also removed unnecessary styles from main.css file

File: src/App.tsx [hunk 3]
```diff
@@ -1,19 +1,11 @@
...
+import { AboutMe } from './components/AboutMe/AboutMe.component.tsx';
 
-import { HeaderComponent } from './components/header/Header.component.tsx';
-
-import './App.css';
+import './components/app/App.css';
 
 function App() {
     return (
         <>
```
Comment (vadym-zinchenko-moc) [inline src/App.tsx]: this is extra Fragment, because you have only one element inside 
`return (
   <AboutMe />
)`

**Review by vadym-zinchenko-moc — CHANGES_REQUESTED** [review]
Screenshot 2024-04-22 at 20 33 14
Screenshot 2024-04-22 at 20 33 24

I don't see that issue is resolved with media requests.

**Comment by vadym-zinchenko-moc** [comment]
Screenshot 2024-04-22 at 20 31 31

why file App.css inside folder app. What the reason for this? this file must be near App component

**Comment by vadym-zinchenko-moc** [comment]
Screenshot 2024-04-22 at 20 43 53

folder images must be inside assets

**Comment by vadym-zinchenko-moc** [comment]
Screenshot 2024-04-22 at 20 43 40

naming must be in one style (if `header` then must be `aboutMe` )

**Comment by NosatskyiK** [comment]
1. Added adaptability, now when the browser window is reduced, the image adapts and becomes smaller
2. Deleted the unnecessary fragment in App.tsx
3. Moved the app.css file to App.tsx
4. Moved the folder with images to the assets folder
5. Renamed the aboutMe file

File: src/components/aboutMe/aboutMe.module.css [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    align-items: center;
+    margin: 10px;
+    gap: 10px;
+  }
+
+  .txt {
+    margin: 10px;
+  }
+
+  .img {
```
Comment (vadym-zinchenko-moc) [inline src/components/aboutMe/aboutMe.module.css]: it's very bad naming for classes `img` `txt `...you have tag `img`. Maybe you can use something more understandable `aboutImage` for example

Comment (vadym-zinchenko-moc) [reply src/components/aboutMe/aboutMe.module.css]: and other classes must be more particular

File: src/components/aboutMe/aboutMe.module.css [hunk 5]
```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap");
+
+body {
```
Comment (vadym-zinchenko-moc) [inline src/components/aboutMe/aboutMe.module.css]: you already added styles for body in App.css file. Why do you use againg. Can you do this in one file?

File: src/components/aboutMe/AboutMe.component.tsx [hunk 6]
```diff
@@ -0,0 +1,42 @@
+import title from '@/assets/img/title.png';
+
+import styles from './aboutMe.module.css';
+
+export const AboutMe = function () {
```
Comment (vadym-zinchenko-moc) [inline src/components/aboutMe/AboutMe.component.tsx]: you can use arrow functions.

File: src/components/aboutMe/AboutMe.component.tsx [hunk 7]
```diff
@@ -0,0 +1,42 @@
+import title from '@/assets/img/title.png';
```
Comment (vadym-zinchenko-moc) [inline src/components/aboutMe/AboutMe.component.tsx]: is it `title`? isn't `about_image` maybe for example?

Comment (vadym-zinchenko-moc) [reply src/components/aboutMe/AboutMe.component.tsx]: are you can name `car`  for example like `bed` for example?? you need to name everything understandable. classes, images etc...

File: src/components/aboutMe/aboutMe.module.css [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+    gap: 10px;
+  }
+
+  .txt {
+    margin: 10px;
+  }
+
+  .img {
+    height: auto;
+    width: 80%;
```
Comment (vadym-zinchenko-moc) [inline src/components/aboutMe/aboutMe.module.css]: Screenshot 2024-04-23 at 11 42 55

in design we image and text in one line.

**Review by vadym-zinchenko-moc — CHANGES_REQUESTED** [review]

**Comment by NosatskyiK** [comment]
1. Changed the names of classes
2. Moved common styles to app.css
3. Replaced the normal function with an arrow function in AboutMe.component.tsx
4. Changed the main image
5. In the mobile version, align text and images on one line

**Review by vadym-zinchenko-moc — APPROVED** [review]

**Pull request merged**